docs/deploy-cephfs.md

```diff
@@ -217,3 +219,20 @@ The Helm chart is located in `charts/ceph-csi-cephfs`.
 **Deploy Helm Chart:**
 
 [See the Helm chart readme for installation instructions.](../charts/ceph-csi-cephfs/README.md)
+
+## Ceph FS Volume Encryption
```

Comment:
Would be better to make a single string "CephFS"

Reply:
:+1: